client/service/test/src/client/mod.rs

```diff
@@ -113,6 +113,7 @@ fn construct_block(
 		Default::default(),
 		&runtime_code,
 		task_executor.clone() as Box<_>,
+		CallContext::Offchain,
```

Comment:
Shouldn't this be `Onchain` since this is supposed to be block construction?

Reply:
I mean it is just some test. :D Not thatttt important :P